# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/ReservedWords.h
# Commit: d9fa28df9787
# Full Hash: d9fa28df9787b4d01135df632c987f20e830b9d6
# Author: Jonatan Klemets <jonatan.r.klemets@gmail.com>
# Date: 2021-12-02 09:42:49
# Regressor Bug: 1736060
# File Overlap Count: 1
# Description:
#   Bug 1736060 - Part 5: Implement import assertions for static imports. r=mgaudet,arai
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D126045
# ==============================================================================

diff -r 5b9622ccd0d2 -r d9fa28df9787 js/src/frontend/ReservedWords.h
--- a/js/src/frontend/ReservedWords.h	Wed Dec 01 18:26:09 2021 +0000
+++ b/js/src/frontend/ReservedWords.h	Wed Dec 01 18:26:10 2021 +0000
@@ -61,6 +61,7 @@
                                                         \
   /* Contextual keywords. */                            \
   MACRO(as, as, TokenKind::As)                          \
+  MACRO(assert, assert_, TokenKind::Assert)             \
   MACRO(async, async, TokenKind::Async)                 \
   MACRO(await, await, TokenKind::Await)                 \
   MACRO(from, from, TokenKind::From)                    \